Sources/ApolloCodegenLib/FileGenerators/EnumFileGenerator.swift

```diff
@@ -1,26 +1,23 @@
 import Foundation
 
 /// Generates a file containing the Swift representation of a [GraphQL Enum](https://spec.graphql.org/draft/#sec-Enums).
-struct EnumFileGenerator: FileGenerator, Equatable {
-  /// The `GraphQLEnumType` object used to build the file content.
-  let enumType: GraphQLEnumType
-  let path: String
-
-  var data: Data? {
-    EnumTemplate(graphqlEnum: self.enumType)
-      .render()
-      .data(using: .utf8)
-  }
-
-  /// Designated initializer.
+struct EnumFileGenerator {
```

Comment:
Nit: We don't need to change this right now, but I usually use caseless enums for objects that are never initialized (only have static functions). That way it's literally impossible to even make an instance of them.